projects/packages/my-jetpack/_inc/components/connection-status-card/connection-issue-tooltip.tsx

```diff
@@ -0,0 +1,52 @@
+import { __ } from '@wordpress/i18n';
+import useMyJetpackConnection from '../../hooks/use-my-jetpack-connection';
+import { InfoTooltip } from '../info-tooltip';
+
+interface AccountError {
+	type: string;
+	message: string;
+	details?: Record< string, unknown >;
+}
+
+/**
+ * Tooltip that displays possible account errors for the current user.
+ *
+ * @return The tooltip component
+ */
+export function ConnectionIssueTooltip() {
+	const { userConnectionData } = useMyJetpackConnection();
+
+	return (
+		userConnectionData.currentUser?.possibleAccountErrors &&
+		Object.keys( userConnectionData.currentUser.possibleAccountErrors ).length > 0 && (
+			<span style={ { display: 'inline-flex', verticalAlign: 'middle' } }>
```

Comment:
Did you mean to leave the inline styles or was this just for testing?

Reply:
It was there before, I merely moved it here.